src/content/docs/ai-observability/advanced-customization/drop-sensitive-data.mdx

```diff
@@ -0,0 +1,306 @@
+
+```
+DIAGRAM HERE
+```
+
+Drop rules let you remove sensitive data before the agent forwards it to NRDB. A drop rule directs the agent to drop certain attributes if the text string within the attribute matches the regex. This lets you still analyze an interaction with your AI assistant without storing your customer's personal data in our databases. 
+
+## Create drop filters [#create-filter]
+
+A single drop rule targets a specified attribute within one event type, but sensitive information from a single AI interaction is stored in multiple events. To ensure that information is completely dropped before it enters NRDB, you need four separate drop rules.
+
+  <img
+      title="Create a drop filter"
+      alt="When you click Create a drop filter, a side modal appears to guide you through dropping text from events and attributes."
+      src={aiDropFilterModal}
+  />
+
+1. Go to **[one.newrelic.com](https://one.newrelic.com) > All capabilities > AI monitoring > Drop filters**, then click **Create drop filter**. 
+1. Create a filter name. Because one kind of data requires four drop rules, we recommend a naming convention that helps you track the events you're dropping data from. 
+1. Paste in the base NRQL query:
+  ```sql
+  SELECT <COLUMN> FROM <EVENT_TYPE> WHERE <COLUMN> RLIKE <REGEX>
+  ```
+1. Referring to the table, update the `<COLUMN>` and `<EVENT_TYPE>` placeholders to match the attributes and events. For example:
+  ```sql
+  SELECT content FROM LlmChatCompletionMessage WHERE content RLIKE <REGEX>
+  ```
+1. Add the regex that corresponds to the kind of information you want to drop. For example, if you're targeting IP addresses, the completed query should read:
+  ```sql
+  SELECT content FROM LlmChatCompletionMessage WHERE content RLIKE ^([0-9]{1,3})\.([0-9]{1,3})\.([0-9]{1,3})\.([0-9]{1,3})$
+  ```
+1. Repeat the above steps to create drop filters for the remaining events and column pairs. 
+
+## Your data compliance responsibilities [#responsibilities]
+
+New Relic can't guarantee that this functionality completely resolves your data disclosure concerns, nor can we provide support for building out your NRQL queries. We recommend that you:
+
+* Review your drop rules and confirm they're accurately identifying and discarding the data you want dropped.
+* Check that your drop rules are still dropping sensitive information after you've created them. 
+
+While drop rules ensure that personal information about your end users aren't stored in NRDB, creating the rules themselves imply the kinds of data you maintain, including the format of your data or systems. This is important when determining control permissions for certain users in your org, as certain permissions let users view and edit all information in the rules you create.
```

Comment:
Is there anything we can link out to from here that they can get more information on? I doubt there is but it seems kind of ominous to not see anything that links out from sections like this. Just a thought!